Directory.Packages.props

```diff
@@ -11,7 +11,7 @@
     <PackageVersion Include="System.Runtime.Extensions" Version="4.3.1" />
     <PackageVersion Include="System.Runtime.InteropServices" Version="4.3.0" />
     <PackageVersion Include="System.Text.Encoding.Extensions" Version="4.3.0" />
-    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.1.5" />
+    <PackageVersion Include="Microsoft.Data.SqlClient" Version="5.2.1" />
```

Comment:
Note: SqlClient 5.1 is LTS, but 5.2 has only "Current" support level. See https://learn.microsoft.com/en-us/sql/connect/ado-net/sqlclient-driver-support-lifecycle

For this reason EF Core also only references SqlClient 5.1

Reply:
We can downgrade it if its a problem.

Reply:
Do downgrade.